electrum/plugin.py

```diff
@@ -162,12 +170,9 @@ def disable(self, name: str) -> None:
 
     @classmethod
     def is_plugin_enabler_config_key(cls, key: str) -> bool:
-        if not key.startswith('use_'):
-            return False
-        # note: the 'use_' prefix is not sufficient to check, there are
-        #       non-plugin-related config keys that also have it... hence:
-        name = key[4:]
-        return name in cls.find_all_plugins()
+        # fixme: the 'use_' prefix is not sufficient to check, there are
+        #         non-plugin-related config keys that also have it.
+        return key.startswith('use_')
```

Comment:
Maybe we should just do a breaking change and change the config keys.
E.g. reserve the prefix "enable_plugin_" for this, instead of "use_".
As part of the change, we could end up (one-time) disabling all plugins for existing users, but I think that's fine.

Reply:
indeed, that's probably the best thing to do

Reply:
Actually, not going to do that.
I think it would be safer to enable plugins not globally, but on a per-wallet basis.
That way, we can write plugin preferences in the encrypted wallet file, and we have some protection against malware.

Reply:
Do you also want this for the built-in plugins?
Also, for some plugins it might not make sense to be per-wallet, e.g. what about plugins that are used as part of the wizard? E.g. consider the trezor plugin. You want to have that enabled when running the wizard if you want to be able to create a wallet containing a trezor.

Reply:
I don't like the idea of having several types of plugins. The initial motivation for the plugin system was to allow third parties to develop their stuff without needing our permission. Maybe some of the current plugins, in particular those that are touching the wizard, should be moved to the main codebase?

Reply:
I am reverting the "plugins per wallet" concept; this makes things too complicated.
thus, I will change the prefix to "enable_plugin_"